int16 msg_id
int16 sender_id
float64 goal_x
float64 goal_y
int16 robot_id
float64 bid
